### PR TITLE
CI - Split TestCafe renovation tests by platforms

### PR DESCRIPTION
**Pull request by AntonSermyazhko**

### CODE REVIEW

**Review by alexserov — DISMISSED** [review]

File: .github/workflows/testcafe_tests.yml [hunk 1]
```diff
@@ -69,7 +69,9 @@ jobs:
           { componentFolder: "editors", name: "editors", quarantineMode: false },
           { componentFolder: "navigation", name: "navigation (1/2)", quarantineMode: false, indices: "1/2" },
           { componentFolder: "navigation", name: "navigation (2/2)", quarantineMode: false, indices: "2/2" },
-          { componentFolder: "renovation", name: "renovation", quarantineMode: false, inProgressRenovation: true }
+          { componentFolder: "renovation", name: "renovation (jquery) (1/2)", platform: "jquery", quarantineMode: false, inProgressRenovation: true, indices: "1/2" },
+          { componentFolder: "renovation", name: "renovation (jquery) (2/2)", platform: "jquery", quarantineMode: false, inProgressRenovation: true, indices: "2/2" },
+          { componentFolder: "renovation", name: "renovation (react)", platform: "react", quarantineMode: false, inProgressRenovation: true }
```
Comment (EugeniyKiyashko) [inline .github/workflows/testcafe_tests.yml]: Where is renovation task for angular platform?

**Review by MikeVitik — DISMISSED** [review]

File: .github/workflows/testcafe_tests.yml (hunk 1 above)
Comment (AntonSermyazhko) [reply .github/workflows/testcafe_tests.yml]: We do not currently have an angular platform in our tests, so I did not add it to the test matrix.

Comment (EugeniyKiyashko) [reply .github/workflows/testcafe_tests.yml]: I am going add tests where tests angular platform soon - https://github.com/DevExpress/DevExtreme/pull/20439. Do I need create additional task there? Or you can add its in current merge?

Comment (AntonSermyazhko) [reply .github/workflows/testcafe_tests.yml]: I added angular platform.
But we have angular tests - it looks like I was wrong :)

**Review by alexserov — APPROVED** [review]

**Review by AryamnovEugeniy — APPROVED** [review]

**Pull request merged**